mkosi/config.py

```diff
@@ -3194,6 +3196,14 @@ def finalize_defaults(namespace: argparse.Namespace, defaults: argparse.Namespac
             hint="Prefix the setting with '@' in the image configuration file to allow overriding it from the command line.", # noqa: E501
         )
 
+    commandline=""
+    for key in cli_valid:
+        value = cli_valid[key]
+        commandline+=f" --{key}={value}"
```

Comment:
The way this is written results in a double space `CLI:  --foob=bar`.
Alternative which eliminates the preceeding space: `" ".join(f"--{key}={value}" for key, value in cli_valid.items())`